### PR TITLE
Add tooltips for sub-category folder

### PR DESCRIPTION
**Pull request by pujiaxun**
Background: I wonder how many problems I have solved. Then I found the tooltip field is available for now. That's it~

### CODE REVIEW

**Comment by jdneo** [comment]
@pujiaxun Sounds great. Would you mind to attach a screenshot in the PR?

**Comment by pujiaxun** [comment]
> @pujiaxun Sounds great. Would you mind to attach a screenshot in the PR?

image

It looks like this. @jdneo

**Comment by jdneo** [comment]
@pujiaxun Just one small comment.

**Comment by pujiaxun** [comment]
```js
        import * as os from "os";
        //...

        return [
            `AC: ${numAC}`,
            `Not AC: ${numNotAC}`,
            `Unknown: ${numUnknown}`,
            `Total: ${problems.length}`,
        ].join(os.EOL);
```

Like this? @jdneo

**Comment by jdneo** [comment]
@pujiaxun Yes, looks much better.

**Comment by pujiaxun** [comment]
@jdneo  Any other suggestions? For instance, I think the name "Not AC" sounds weird...

**Comment by jdneo** [comment]
@pujiaxun hmm, good question.

Regarding the wording. My suggestion is `Not AC` -> `Failed`

Either `Unknown` or `Total`, one of them can be removed I think? What do you think?

**Comment by pujiaxun** [comment]
@jdneo I prefer to keep `Total` if I have to give up one of them. After all, the tag `Unknown` is somewhat confusing.

**Comment by jdneo** [comment]
Same to me. So let's keep `AC`, `Failed` and `Total`.

File: src/explorer/LeetCodeTreeDataProvider.ts [hunk 1]
```diff
@@ -180,7 +180,6 @@ export class LeetCodeTreeDataProvider implements vscode.TreeDataProvider<LeetCod
 
         let numAC: number = 0;
         let numNotAC: number = 0;
```
Comment (jdneo) [inline src/explorer/LeetCodeTreeDataProvider.ts]: nit:
`numAC` -> `acceptedNum`
`numNotAC` -> `failedNum`

Comment (pujiaxun) [reply src/explorer/LeetCodeTreeDataProvider.ts]: cool

**Pull request merged**